Dockerfile

```
@@ -0,0 +1,52 @@
+FROM ubuntu:xenial as base
+
+# Install system requirements
+RUN apt update && \
+    # Global requirements
+    DEBIAN_FRONTEND=noninteractive apt install -y language-pack-en git build-essential software-properties-common curl git-core libxml2-dev libxslt1-dev libmysqlclient-dev libxmlsec1-dev libfreetype6-dev libssl-dev swig gcc g++ \
+    # openedx requirements
+    gettext gfortran graphviz libgraphviz-dev libffi-dev libfreetype6-dev libgeos-dev libjpeg8-dev liblapack-dev libpng-dev libsqlite3-dev libxml2-dev libxmlsec1-dev libxslt1-dev ntp pkg-config python3.5 python3-pip python3-dev \
```

I would suggest that you sort packages alphabetically with one package per line so that it's easier to read/maintain, _e.g._:

```suggestion
    # [...]
    gettext \
    gfortran \
    graphviz \
    # [...]
```

---

Will do, thank you!